evalml/pipelines/components/utils.py

```diff
@@ -56,7 +56,19 @@ def allowed_model_families(problem_type):
     return list(set([e.model_family for e in estimators]))
 
 
-def get_estimators(problem_type, model_families=None, excluded_model_families=None):
+def _filter_multiseries_estimators(estimators, is_multiseries):
+    if is_multiseries:
+        return [estimator for estimator in estimators if estimator.is_multiseries]
```

Comment:
nit and maybe for a follow up: could `estimator.is_multiseries` be something like `estimator.supports_multiseries`? Think its more clear especially since we're passing `is_multiseries` everywhere for now.